[Benchmark] Adding MVTamperBench

amitbcp opened the pull request:
1. Paper : https://arxiv.org/pdf/2412.19794

srikant86panda committed:
MVBench data prep fixes

srikant86panda committed:
handle missing video file with mvbench

srikant86panda committed:
MVTamperBench base

FangXinyu-0913 committed:
merge main and add exception information in generate tsv

FangXinyu-0913 committed:
add nturgb-d information to generate tsv function

srikant86panda committed:
sample dataset

srikant86panda committed:
updated tamperbench

srikant86panda committed:
updated md5 and dataset name for complete dataset

amitbcp committed:
Merge branch 'open-compass:main' into main

Commit:
resolve conflict and update with VLM

srikant86panda committed:
fix on moving perception file to correct folder

srikant86panda committed:
Merge branch 'MVTamperBench' of https://github.com/srikant86panda/VLMEvalKit into MVTamperBench

srikant86panda committed:
adding correct md5 for total ~19k sample

Commit:
Local changes

Commit:
Merge remote-tracking branch 'origin/MVTamperBench' into MVTamperBench

amitbcp committed:
Merge branch 'open-compass:main' into main

Commit:
Local Changes to run

Commit:
resolve merge conflict

srikant86panda committed:
updated evaluation

srikant86panda committed:
updated to latest change

srikant86panda committed:
incroporated metrics and pre processing change

srikant86panda committed:
sample file support

srikant86panda committed:
reporting fix

srikant86panda committed:
fix with md5 check

srikant86panda committed:
updated md5 and dataset support

srikant86panda committed:
ntu data path fix

srikant86panda committed:
dataset support added on run.py

srikant86panda committed:
ntu files removed

srikant86panda committed:
md5 updated

srikant86panda committed:
run file updated

srikant86panda committed:
conflict resolved with main branch

srikant86panda committed:
resolve conflict cgbench_dataset

ppattnayak committed:
minor formating

ppattnayak committed:
adding explanation to check_ans

ppattnayak committed:
adding explanation to check_ans_with_model

ppattnayak committed:
updated build_prompt with comments

ppattnayak committed:
adding messages to check integrity method

ppattnayak commented on vlmeval/dataset/tamperbench.py:
```diff
@@ -0,0 +1,460 @@
+        else:
+            repo_id = f'Srikant86/{dataset_name}'
+
+        def check_integrity(pth):
```
can't this be optimized instead of this nested function?

bhargavak1409 committed:
Added torchvision to dependencies

bhargavak1409 committed:
removed unused imports

bhargavak1409 committed:
Refactor tamperbench.py: improve code readability and add detailed docstrings for methods

amitbcp committed:
Add metrics code for aggregation

amitbcp committed:
Fix imports

amitbcp committed:
Save results

amitbcp committed:
Add base dataset name

amitbcp committed:
function parameter

Commit:
ORD changes

Commit:
Merge branch 'MVTamperBench_updated' of https://github.com/srikant86panda/VLMEvalKit into MVTamperBench_updated

amitbcp committed:
remove across dataset aggregation

Commit:
Merge branch 'MVTamperBench_updated' of https://github.com/srikant86panda/VLMEvalKit into MVTamperBench_updated

amitbcp committed:
Fix linting

amitbcp committed:
Updated pre-commit check

ChuanyangZheng committed:
[Model] Add BailingMM-Pro-0120 (#740)

lihytotoro committed:
[Fix] Fix num_beams for MiniCPM-o-2_6 (#745)

Co-authored-by: Poppy Xu <xuyingjing1@modelbest.cn>

kennymckormick committed:
[Minor] Change Arg Name: "--api_nproc" -> "--api-nproc" (#746)

OliverLeeXZ committed:
[Model] Add emu3-chat and emu3-gen (#738)

* add emu3-chat and emu3-gen

* update emu

---------

Co-authored-by: kennymckormick <dhd.efz@gmail.com>

niboshi committed:
[Minor] Add judge args option (#736)

kennymckormick committed:
[Minor] "Emu3" -> "emu3"

jamari-morrison committed:
[Improvement] Moondream2 new release update (#742)

PhoenixZ810 committed:
Refine Lmdeploy with InternVL2.5-78B-MPO & QVQ-72B-preview (#749)

* change config

* modify LmdeployWrapper

* modify build_mpo_prompt.py with WeMath

iyuge2 committed:
[Model] Update glmvision (#748)

* update glm vision

* update glm vision

* fix

* update glm4v_plus

* update glmvision to mass api

kennymckormick committed:
[Minor] Change Arg Name: "--api_nproc" -> "--api-nproc" (#750)

kennymckormick committed:
Update run.py

kennymckormick committed:
[Minor] Fix GLM-V Import

DataWizardLiu committed:
[Model] Doubao VL (#747)

* add doubao_vl_api.py

add doubao_vl api

* Update config.py

* Update __init__.py

* Update config.py

kennymckormick committed:
[Fix] Fix Emu3 Inference

andimarafioti committed:
[Model] Add SmolVLM-256M and SmolVLM-500M and fix prompts for benchmarks (#751)

lrlbbzl committed:
[Model] Add URSA-8B (#737)

* update ursa

* fix ursa

* fix ursa

* update ursa final

---------

Co-authored-by: 骆瑞霖 <luoruilin@bytedance.com>

PhoenixZ810 committed:
change WeMath file & build_prompt (#755)

PhoenixZ810 committed:
fix wemathv2 evaluate, move ursa import to init (#757)

* change WeMath file & build_prompt

* fix wemath evaluate

* move ursa import to init

FangXinyu-0913 committed:
[Fix] sklearn Import on dataset/utils/tamperbench

FangXinyu-0913 committed:
[Fix] fix on private data_path import

MMoshtaghi committed:
[Fix] Fixed the cache_path in pixtral.py (#756)

PhoenixZ810 committed:
fix claude_wrapper with encode_image&official backend (#758)

PhoenixZ810 committed:
Claude (#759)

* fix claude_wrapper with encode_image&official backend

* fix claude encode

* fix fmp to fmt

Commit:
Merge branch 'MVTamperBench_updated' of https://github.com/srikant86panda/VLMEvalKit into MVTamperBench_updated

PhoenixZ810 committed:
Claude refine .jpg (#760)

* fix claude_wrapper with encode_image&official backend

* fix claude encode

* fix fmp to fmt

* .jpg

* refine .jpg

* fix WeMath eval model

kennymckormick committed:
[Model] Support Qwen-2.5-VL and Janus-Pro-7B (#762)

* update .gitignore

* support Qwen-2.5-VL

* support Janus-Pro-7B

kennymckormick committed:
add Step1o

kennymckormick committed:
[Improvement] Refine reuse strategy & implement sympy timeout (#764)

* [Fix] Better Import

* [Improvement] Also reuse auxiliary files by default

* [Fix] Fix fetch_aux_files

* update

* implement timeout for sympy usages

kennymckormick committed:
[Fix] Fix minor issues for llava 1.5

PhoenixZ810 committed:
add mathverse_visiononly_cot (#765)

MMoshtaghi committed:
Updated the generate in LLaVA-OneVision-HF (#761)

* Fixed the cache_path in pixtral.py

* Updated response in llava.py

only return the model's generated response instead of the whole output

* Removed the unnecessary prompt print in LLaVA-OneVision-HF

kennymckormick committed:
[Minor] Support cmd `vlmutil scan`

kennymckormick committed:
[Benchmark] Support MMMU Pro (#768)

* support MMMU_Pro

* support MMMU_Pro as concat dataset

* [Fix] Fix Bug

* Fix Bug

* [Fix] Fix MMMU_Pro_V judge

* Fix vlmeval tools: EVAL

* Bug Fix

* [Fix] Fix MMMUPro COT evaluation

* fix

* Fix

* Fix

dongyh20 committed:
[Model] Add Ola Model (#752)

* add ola model

* Update ola_model.py

* Update builder.py

kennymckormick committed:
[Minor] Update SiliconFlowAPI

kennymckormick committed:
[Minor] Fix SiliconFlowAPI

kennymckormick committed:
[Minor] Adopt CircularEval for all datasets with 'circular' in names

kennymckormick committed:
[Improvement] Better Implementation for CircularEval (#770)

* update .gitignore

* update CIRCULAR implementation

* [Refine] Update CircularEval

PhoenixZ810 committed:
Claude Timeout (#771)

* add ClaudeWrapper Timeout, add Claude3-5V_Sonnet_20241022_tem07

FangXinyu-0913 committed:
[Fix] Fix build dataset from config for image dataset

PhoenixZ810 committed:
Add Dynamath_noprompt (#776)

* add ClaudeWrapper Timeout, add Claude3-5V_Sonnet_20241022_tem07

* add dynamath_noprompt

Commit:
Removed debug prints

marvinzh committed:
[Fix] Move barrier to the starting point of the for-loop (#772)

current implement will not work if user use infer only mode under multiple processes

runninglsy committed:
add Ovis2

runninglsy committed:
remove redundant parentheses

Mor-Li committed:
Merge pull request #782 from runninglsy/main

[Model] Add Ovis2

kennymckormick committed:
Update SEEDBench2_Plus MD5

zenjieli committed:
[Minor] Fix bug in extracting dataset_name from eval file name (#778)

Co-authored-by: zli <zli@mail.com>

LingyiHongfd committed:
update worlsense (#775)

Co-authored-by: FangXinyu-0913 <fangxinyutju202009@126.com>

kq-chen committed:
[Fix] qwen2.5vl evaluation do not use custom prompt (#786)

kennymckormick committed:
[Improvement] Support ImageSimpleQA data class and CreationBench data class (#790)

* support mmbench_c

* Fix BUG

* update mmbench_c

* update SenseChat MaxTokens

* Use Dual Eval for MMBench_C by default

* [Fix] Fix MMBench-C

* [Fix] Update Max_Token Args

* [Fix] Fix Bug

* [Fix] Fix Circular

* update

* update

* update config

* update dlist

* Update Config

* Update dataset name

* add group live

* [Minor] Set max_tokens to 2048

* [Minor] Change the max_tokens to 2048 for models that will be evaluated on LiveMMBench

* remove print

* patch error

* fix md5

* [Minor] Update mmqa_display

* Fix

* minor

PhoenixZ810 committed:
Update README and QUICKSTART (#795)

* change readme and quickstart

* refine readme and quickstart.

* refine readme_cn.md

PhoenixZ810 committed:
New document (#796)

* change readme and quickstart

* refine readme and quickstart.

* refine readme_cn.md

* modify date

PhoenixZ810 committed:
Mmalignbench (#805)

* add mmalignbench

* add judge model gpt4o

kennymckormick committed:
[Improvement] Refine Prompts for ShortQA Evaluation (#808)

yaogang2060 committed:
[Improvemnet] Remove Unnecessary vlmeval/vlm/internvl_chat.py  (#792)

#657 has deleted this file, while #729 added it back, which is not needed.😢

FangXinyu-0913 committed:
Merge branch 'main' into MVTamperBench_updated
